Makefile

```diff
@@ -32,8 +32,8 @@ distclean: clean
 download/leaflet-$(DEP_LEAFLET_VERSION).tar.gz:
```

Comment:
You have to change the file name suffix in the make target because the "produced" file ends with ".zip".